test/integration/scheduler/queueing/queue.go

```diff
@@ -76,6 +78,8 @@ type CoreResourceEnqueueTestCase struct {
 	// EnableSchedulingQueueHint indicates which feature gate value(s) the test case should run with.
 	// By default, it's {true, false}
 	EnableSchedulingQueueHint sets.Set[bool]
+	// SchedulerProfile is the scheduler profile overriding the default scheduler profile.
+	SchedulerProfile *configv1.KubeSchedulerProfile
```

Comment:
Then, can we change it to `EnabledPlugins []string`?

Reply:
I think we could do that, but I left it highly flexible.

Reply:
I prefer my suggestion. I do not see a future that we need to override anything other than enabled plugin set.

Reply:
@sanposhiho
We discussed that other test cases also specify related plugins. For this reason, I think it is better to keep this.

Reply:
Make it `EnablePlugins` if it does not seem necessary in other test cases +1

Reply:
Yes, and are you saying those test cases could not set `EnabledPlugins` somehow?

Reply:
I honestly don't know because I haven't tried it yet. Well, it's easy to change when we need the plugin parameters, so let's do it in the minimal form you suggest first! Thanks for the suggestion, as I was wondering here too.

Reply:
Yup, we don't meed that unless we implement some QHint changing its behavior with the plugin config.

Reply:
I've updated it as you suggested

Reply:
@sanposhiho Have a happy new year! 🎍